charts/zigbee2mqtt/values.yaml

```
@@ -76,16 +81,48 @@ config:
     # MQTT server authentication, uncomment if required:
     # user: my_user
     # password: my_password
+    # client_id: my_id
+    # Alternatively, credentials may be put into a separate file, managed through a secret:
+    # password: '!secret password'
+
+    # Optional: Include device information to mqtt messages (default: false)
+    include_device_information: true
 
   # USB / Serial settings
   serial:
     # Location of your zigbee device
-    port: /dev/ttyACM0
+    port: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2400981-if00
+
+    # Optional: adapter type, not needed unless you are experiencing problems (options: zstack, deconz)
+    # adapter: deconz
 
   advanced:
     network_key: GENERATE
     log_output:
       - console
+    log_level: info
+
+    # Optional: ZigBee channel, changing requires re-pairing of all devices. (Note: use a ZLL channel: 11, 15, 20, or 25 to avoid Problems)
+    channel: 11
+
+    # Optional: Baudrate for serial port (default: 115200 for Z-Stack, 38400 for Deconz)
+    # baudrate: 38400
+
+    # Optional: RTS / CTS Hardware Flow Control for serial port (default: false)
+    # rtscts: true
+
+    # Optional: Add a last_seen attribute to MQTT messages, contains date/time of last Zigbee message
+    # possible values are: disable (default), ISO_8601, ISO_8601_local, epoch (default: disable)
+    last_seen: 'ISO_8601'
+
+    homeassistant_discovery_topic: 'homeassistant'
+    homeassistant_status_topic: 'homeassistant/status'
+
+    # Optional: Enables report feature (see information -> report for more details) (default: false)
+    report: true
```

```suggestion
    # report: true
```
I suggest commenting this out, since it's offered as an optional config parameter.